zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -1107,7 +1107,7 @@ cfu.warn.nolaunch = The latest ZAP release: {0} has been downloaded to\n{1}\nYou
+cmdline.help = cmdline.help = Usage:\n\t{0} [Options]\nCore options:\n\t-version                 Reports the ZAP version\n\t-cmd                     Run inline (exits when command line options complete)\n\t-daemon                  Starts ZAP in 'daemon' mode, i.e. without a UI\n\t-config <kvpair>         Overrides the specified 'key=value' pair in the configuration file\n\t-configfile <path>       Overrides the 'key=value' pairs with those in the specified properties file\n\t-dir <dir>               Uses the specified directory instead of the default one\n\t-installdir <dir>        Overrides the code that detects where ZAP has been installed with the specified directory\n\t-h                       Shows all of the command line options available, including those added by add-ons\n\t-help                    The same as -h\n\t-newsession <path>       Creates a new session at the given location\n\t-session <path>          Opens the given session after starting ZAP\n\t-lowmem                  Use the database instead of memory as much as possible - this is still experimental\n\t-experimentaldb          Use the experimental generic database code, which is not surprisingly also still experimental\n\t-nostdout                Disables the default logging through standard output\n\t-loglevel <level>        Sets the log level, overriding the values specified in the log4j2.properties file in the home directory\n\t-sbomzip <path>          Creates a zip file containing all of the available SBOMs\n\t-suppinfo                Reports support info to the command line and exits\n\t-silent                  Ensures ZAP does not make any unsolicited requests, including 'check for updates'\nAdd-on options:\n
```

Comment:
Just noticed the duplicated resource name (`cmdline.help =`).

Reply:
Good catch!